README.md

```diff
@@ -22,6 +22,16 @@ The only required input is `project-name`.
    that CodeBuild requires.
    By default, the action uses the buildspec file location
    that you configured in the CodeBuild project.
+1. **compute-type-override** (optional) :
+   The name of a compute type for this build that overrides the one specified
+   in the build project.
+1. **environment-type-override** (optional) :
+   A container type for this build that overrides the one specified in the
+   build project.
+1. **image-override** (optional) :
+   The name of an image for this build that overrides the one specified
+   in the build project. By default, the action uses the default environment
```

Comment:
nit pick. Is the explanation here meant to match the public documentation? If so, I don't see the second sentence in our docs.

Reply:
Resolved